index.html

```diff
@@ -26,7 +26,7 @@
       <header>
```

Comment:
- [ ] The container of the logo and hamburger must be the header, not a nav tag
- [ ] And the hamburger button must be wrapped for a `nav` tag, check [this link](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/nav)